doc/test_with_td_payload.md

```diff
@@ -34,7 +34,7 @@ $ cd ..
 ### Generate final.bin
 ```
 $ cargo xbuild -p td-shim --target x86_64-unknown-uefi --release --features=main,tdx --no-default-features
-$ cargo run -p td-shim-tools --bin td-shim-ld -- target/x86_64-unknown-uefi/release/ResetVector.bin target/x86_64-unknown-uefi/release/td-shim.efi target/x86_64-unknown-uefi/release/test-td-payload.efi -o target/x86_64-unknown-uefi/release/final-pe.bin
+$ cargo run -p td-shim-tools --bin td-shim-ld --no-default-features --features="linker" -- target/x86_64-unknown-uefi/release/ResetVector.bin target/x86_64-unknown-uefi/release/td-shim.efi target/x86_64-unknown-uefi/release/test-td-payload.efi -o target/x86_64-unknown-uefi/release/final-pe.bin
```

Comment:
Why do we need "--no-default-features" here?

Reply:
It is specific for QEMU boot, I will remove it.